button.js

```diff
@@ -16,12 +14,9 @@ function replaceWord() {
     },
   })
     .then((response) => {
-      console.log(response);
-      response.json().then(() => {
-        /* 
-          Complete this handler function with code that populates an html 
-          element with the random word 
-        */
+        response.json().then((responseBody) => {
+          console.log(responseBody.word);
```

Comment:
In my example I used console.log statements to show what the structure of responseBody is. Once we're past the need for that example, we should remove console.log statements from the code.